build.sbt

```diff
@@ -2,10 +2,10 @@ import com.typesafe.sbt.SbtGhPages.ghpages
 import com.typesafe.sbt.SbtGit.git
 import de.heikoseeberger.sbtheader.HeaderPattern
 
-lazy val buildSettings = Seq(
+lazy val commonSettings = Seq(
   organization := "com.fortysevendeg",
```

Comment:
I'll leave this in place here and add a ticket to address this before any future releases